docs/topics/commands.rst

```diff
@@ -368,11 +376,31 @@ given. Also supports UNIX-style local file paths, either relative with
 ``./`` or ``../`` prefixes or absolute file paths.
 See :ref:`topics-shell` for more info.
 
+Supported options:
+
+* ``--spider=SPIDER``: bypass spider autodetection and force use of specific spider
+
+* ``-c code``: evaluate the code in the shell, print the result and exit
+
+* ``--no-redirect``: do not follow HTTP 3xx redirects (default is to follow them)
```

Comment:
Ah, ok. Yes, --no-redirect only affects the redirect for "url" in `scrapy shell --no-redirect url`. Once, Inside the shell, `>>> fetch(url)` does follow redirects (with this patch applied).
Definitely worth mentioning. I haven't found good wording yet (I already struggled writing this comment)